doctor/models.py

```
@@ -0,0 +1,595 @@
+        ('orthopedics', 'ارتوپدی'),
+        ('ophthalmology', 'چشم'),
+        ('ent', 'گوش، حلق و بینی'),
+        ('gynecology', 'زنان'),
+        ('urology', 'اورولوژی'),
+        ('emergency', 'اورژانس'),
+        ('anesthesiology', 'بیهوشی'),
+        ('radiology', 'رادیولوژی'),
+        ('pathology', 'آسیب‌شناسی'),
+        ('surgery', 'جراحی عمومی'),
+        ('other', 'سایر')
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)